### ISSUE TITLE
Technically every sunday is a holiday in `DE-HE`

### ISSUE THREAD

**Issue by leipert** [commenthol/date-holidays#14]
This is not fixed in #12:

Technically every sunday is a holiday in `DE-HE`.

>Public holidays are the **sundays** and [...]
Source: http://www.datumsrechner.de/gesetze/Hessen.pdf

I do not know how you want to handle this.

**Comment by commenthol** [comment]
Sorry, there is no rule for that yet. I don't know if this is really of relevance in this lib as it would "pollute" the real information a user would like to see. Legally seen the it would be correct to show each Sunday in the list. I am undecided in this case.

**Comment by Maybach91** [comment]
I agree with @commenthol that it would be a kind of pollution. But i need the option to declare the sundays as holiday too. Or just to check if the selected date is a sunday. Is there any option for this?

**Comment by commenthol** [comment]
@Maybach91 No there is no such option. As you are checking date by date I would recommend to use the `new Date(dateStr).getDay() === 0` comparison instead of checking the localized string.
